database/init_db.go

```diff
@@ -50,26 +50,29 @@ func main() {
 
 	// SQL instruction to create a table.
 	sqlStatement := `
-	CREATE TABLE IF NOT EXISTS public.test_objects (
+	CREATE TABLE IF NOT EXISTS public.resource (
 		"id" serial PRIMARY KEY,
 		"api_version" varchar NOT NULL,
+		"cluster" varchar NOT NULL,
+		"cluster_uid" uuid NOT NULL,
```

Comment:
Since these are marked as `NOT NULL` the sink will need to provide these values when writing a resource to the database. My concern is that I'm not entirely sure how to get these values